### ISSUE TITLE
Error : to make it work using sveltekit 

### ISSUE THREAD

**Issue by borisrose** [vatro/svelthree#211]
Hello I am currently trying to use one example in my project and your lib does not work at all. 

as if the **type** keyword created an error along the way.

<img width="1060" alt="Capture d’écran 2023-03-28 à 14 11 09" src="https://user-images.githubusercontent.com/69965863/228232427-6c3b3273-00df-45ef-971f-5b93be34cc98.png">

<img width="929" alt="Capture d’écran 2023-03-28 à 14 12 32" src="https://user-images.githubusercontent.com/69965863/228232447-337d471c-0729-4daa-9e40-b45bd1133006.png">

<img width="1013" alt="Capture d’écran 2023-03-28 à 14 15 00" src="https://user-images.githubusercontent.com/69965863/228232997-264671d0-b616-4a6e-8044-6b99a112108f.png">

for that project, I use sveltekit, prisma et typescript

**Comment by engageintellect** [comment]
Agreed. I can get the examples to work in a REPL. But in a dev env everything breaks upon refreshing the page.

**Comment by vatro** [comment]
Hello @borisrose , Hello @engageintellect!

⚠️ **Don't use the version published on npm!** (very old /  the very first proof of concept)
⚠️ **`svelthree` is not stable / still experimental / not production-ready**

If you want to try it anyway 👉 look at [**README**](https://github.com/vatro/svelthree/blob/dev/README.md#how-to-try-latest-svelthree-now-) on **how to try latest (_not published via npm_) `svelthree` NOW**. The "current" version is `svelthree-1.0.0-next.0.98`. The upcoming version `svelthree-1.0.0-next.1` will be the fairly stable one and more open to contributing, including CI/CD, tests, docs, examples etc. I'm on it (_though I didn't have much time lately_), see e.g. currently active development branches: [**204-refactor-svelthreeinteraction**](https://github.com/vatro/svelthree/tree/204-refactor-svelthreeinteraction) and [**204-implement-wheelevent**](https://github.com/vatro/svelthree/tree/204-implement-wheelevent)

☝️ Meanwhile, you might take a look at / try [**threlte**](https://github.com/threlte/threlte)!  (_if not already done so_)

Sorry for the inconvenience! 😬

**FYI**: I've just deprecated all published (_very old, outdated_) `svelthree`-versions, means the next one published will be `svelthree-1.0.0-next.1` aiming to be stable for any further development / usage.

**Comment by engageintellect** [comment]
@vatro 

Thank you very much for following up on this. I'm looking forward to trying out the items you mentioned. I really love the idea of this project. 

Thanks again!
